# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../glibc/sparc-gdb_fix.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- glibc-2.3.2/sysdeps/unix/sysv/linux/sparc/sys/ucontext.h.orig	2003-08-03 19:45:12.000000000 +0000
+++ glibc-2.3.2/sysdeps/unix/sysv/linux/sparc/sys/ucontext.h	2003-08-03 19:47:29.000000000 +0000
@@ -19,6 +19,8 @@
 #ifndef _SYS_UCONTEXT_H
 #define _SYS_UCONTEXT_H	1
 
+#ifdef __GLIBC__
+
 #include <features.h>
 #include <signal.h>
 #include <bits/wordsize.h>
@@ -267,4 +269,5 @@
   } ucontext_t;
 
 #endif /* __WORDSIZE == 32 */
+#endif /* __GLIBC__ */
 #endif /* sys/ucontext.h */
